src/utils/generateExplorerLink.ts

```diff
@@ -0,0 +1,24 @@
+import { AddEthereumChainParameter } from '@web3-react/types';
+
+import { WalletType } from '../web3/connectors';
+
+export function generateExplorerLink(
```

Comment:
May be let's do as part of transaction selector? In that case it should have access to activeWallet and getChainParameters